packages/graphile-build-pg/src/plugins/PgMutationCreatePlugin.js

```diff
@@ -44,6 +57,16 @@ export default (function PgMutationCreatePlugin(
           .filter(table => !!table.namespace)
           .filter(table => table.isSelectable)
           .filter(table => table.isInsertable)
+          .filter(
+            table =>
+              pgEnableDefaultMutationTables.length == 0 ||
+              pgEnableDefaultMutationTables.some(function(element) {
```

Comment:
I think `pgEnableDefaultMutationTables  = []` should be equivalent to `pgDisableDefaultMutations = true` - it's a whitelist with no entries.

As such, this check should be more like: